core/templates/components/summary-tile/topic-summary-tile.component.ts

```diff
@@ -80,6 +87,12 @@ export class TopicSummaryTileComponent {
 
     return '#' + newRValue + newGValue + newBValue;
   }
+
+  // Remove this method when translation service is extended.
+  hasTranslationKey(): boolean {
+    return this.i18nLanguageCodeService.hasTranslations(
```

Comment:
Why "translations" (plural)?

Also what is the conceptual difference between "hasTranslationKey" and "hasTranslation(s)"? If they are the same thing (as is suggested by this function) then why are the names different?